docs/jep/11859-type-annotations.md

```diff
@@ -0,0 +1,160 @@
+# Type Annotation Roadmap for JAXS
+_jakevdp@_
+_Aug 2022_
+
+## Background
```

Comment:
Nit: Maybe add spacing after `#`/`##` section titles here and across the doc (ref: https://google.github.io/styleguide/docguide/style.html#add-spacing-to-headings)

For example:

```suggestion
## Background

```